LoRaEngine/modules/LoRaBaseStationModule/start_basestation_amd64.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ -z "$RESET_PIN" ]]; then
+    echo "No RESET_PIN environment variable set, skipping the pin reset. If you experience problem with starting the concentrator please set this variable to your manufacturer reset pin"
+else
+    echo "Resetting the pin"
+    ./reset_lgw.sh stop $RESET_PIN
+    ./reset_lgw.sh start $RESET_PIN
+    echo "Finished resetting the pin"
+fi
+
+
+if [[ -z "$SPI_DEV" ]] || [[ $SPI_DEV == '$LBS_SPI_DEV' ]]; then
+    echo "No custom SPI dev set up, defaulting to spi dev 0"
+    SPI_DEV = 0
+fi
+
+if [[ -z "$TC_URI" ]]; then
```

Comment:
So far we only encountered device with this setting on the ARM architecture. I can't exclude it might be needed in the past, but [our current packet forwarder implementation doesn't support it](https://github.com/Azure/iotedge-lorawan-starterkit/blob/dev/LoRaEngine/modules/LoRaWanPktFwdModule/Dockerfile.amd64) therefore I would suggest to keep it as is.